modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,51 @@
+:experimental:
+= Migrating from CoreOS Container Linux (CL) to Fedora CoreOS (FCOS)
+
+Fedora CoreOS is the successor of CoreOS Container Linux, and current users of CL are encouraged to migrate to FCOS. This page documents the differences between CL and FCOS to ease the transition to FCOS.
+
+:toc:
+
+== Introduction
+
+To migrate from CL to FCOS, you must convert your old Container Linux Configs, Ignition configs, or `cloud-config` files to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration details have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
+
+== Installation changes
+
+The following changes have been made to the installation process:
+
+* The `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`], which offers similar functionality.
+* The `autologin` kernel command-line parameter is not currently supported in FCOS.
+* Certain CL platforms, such as Vagrant, are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
+
+== Configuration changes
+
+When writing FCC files, keep the following changes in mind:
+
+* Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
```

Comment:
That was true for CLCs also.  Not sure why this is here?

Reply:
Good question. I'm removing it.